arch/x86/entry.S

```diff
@@ -180,9 +180,10 @@ ENTRY(enter_usermode)
     SAVE_ALL_REGS
```

Comment:
Please squash this whole commit with https://github.com/KernelTestFramework/ktf/pull/290/commits/b7a0639636a5413363e4250f2aa9802dcc91347c